src/proof/dot/dot_printer.cpp

```diff
@@ -46,23 +46,40 @@ std::string DotPrinter::sanitizeString(const std::string& s)
   return newS;
 }
 
+void DotPrinter::countSubproofs(
```

Comment:
This is a tree traversal, are you sure you want to do this?

Reply:
For now, yes. It's like that by design. Eventually it'll be optional.

Reply:
Should maybe make it non-recursive here at least.

Reply:
Changed to that. I already made it a DAG traversal as well, it was very weird otherwise, I think I was just confused before. :)